service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderController.java

```diff
@@ -265,16 +263,15 @@ public CallbackResponse handleAboutToSubmit(@RequestBody CallbackRequest callbac
             data.put("state", State.CASE_MANAGEMENT);
             removeTemporaryFields(caseDetails, "sdoRouter");
 
-            if (caseData.isSendingNoticeOfProceedings() && featureToggleService.isSendNoticeOfProceedingsFromSdo()) {
-                List<DocmosisTemplates> docmosisTemplateTypes =
-                    caseData.getNoticeOfProceedings().mapProceedingTypesToDocmosisTemplate();
+            List<DocmosisTemplates> docmosisTemplateTypes =
```

Comment:
is it equivalent? What about if(caseData.isSendingNoticeOfProceedings())

Reply:
It was possibly no longer needed in the first place. Feature toggle was probably enough. While toggled on sending NOP's is now mandatory, hence why `caseData.isSendingNoticeOfProceedings()` has been removed.

Reply:
leaving as is for now then